src/metamask/index.ts

```diff
@@ -0,0 +1,36 @@
+import { Page } from "puppeteer";
+
+import { Dappeteer } from "..";
+
+import { addNetwork } from "./addNetwork";
+import { approve } from "./approve";
+import { confirmTransaction } from "./configrmTransaction";
+import { importPk } from "./importPk";
+import { lock } from "./lock";
+import { sign } from "./sign";
+import { switchAccount } from "./switchAccount";
+import { switchNetwork } from "./switchNetwork";
+import { unlock } from "./unlock";
+
+export type SetSignedIn = (state: boolean) => Promise<void>;
+export type GetSingedIn = () => Promise<boolean>;
+
+export const getMetamaskMethods = async (page: Page, version?: string): Promise<Dappeteer> => {
```

Comment:
```suggestion
export const getMetamask= async (page: Page, version?: string): Promise<Dappeteer> => {
```

Reply:
`getMetamask` is in the root that why I named it `getMetamaskMethods` and this function only return methods when the root find MetaMask window and call this function to get methods for window